relay-server/src/actors/events.rs

```diff
@@ -769,12 +804,14 @@ impl EventProcessor {
         let apple_crash_report_attachment = envelope
             .get_item_by(|item| item.attachment_type() == Some(AttachmentType::AppleCrashReport));
 
+        let mut event = state.event.get_or_insert_with(Event::default);
```

Comment:
Move this into the `if let Some()` because we must not create the event payload if there is no attachment.